--- Koord.entitlements	2023-01-11 14:08:02.101330800 +0200
+++ Koord_appstore.entitlements	2023-01-11 14:09:02.142784800 +0200
@@ -12,5 +12,11 @@
 	<true/>
 	<key>com.apple.security.files.user-selected.read-write</key>
 	<true/>
+	<key>com.apple.application-identifier</key>
+ 	<string>TXZ4FR95HG.live.koord.Koord-RT</string>
+	<key>com.apple.security.application-groups</key>
+	<string>TXZ4FR95HG.koord</string>
+ 	<key>com.apple.developer.team-identifier</key>
+ 	<string>TXZ4FR95HG</string>
 </dict>
 </plist>
